.github/workflows/cherry-pick.yml

```
@@ -0,0 +1,128 @@
+name: Cherry pick commits from PR
+on:
+  issue_comment:
+    types:
+      - created
+jobs:
+  # Basic information extraction and validation
+  pr_info:
+    if: |
+      github.event.issue.pull_request &&
+        startsWith(github.event.comment.body, '/cherry-pick ') &&
+        (github.event.comment.author_association == 'MEMBER' ||
+         github.event.comment.author_association == 'OWNER')
+
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      pull-requests: write
+    outputs:
+      status: ${{ steps.pr_info.outputs.status }}
+      base_ref: ${{ steps.pr_info.outputs.base_ref }}
+      merge_commit: ${{ steps.pr_info.outputs.merge_commit }}
+    env:
+      GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+    steps:
+      - name: Extract PR information
+        id: pr_info
+        run: |
+          echo "status=$(gh pr view ${{ github.event.issue.number }} --repo ${{ github.repository }} --json state --jq '.state')" >> $GITHUB_OUTPUT
+          echo "base_ref=$(gh pr view ${{ github.event.issue.number }} --repo ${{ github.repository }} --json baseRefName --jq '.baseRefName')" >> $GITHUB_OUTPUT
+          echo "merge_commit=$(gh pr view ${{ github.event.issue.number }} --repo ${{ github.repository }} --json mergeCommit --jq '.mergeCommit.oid')" >> $GITHUB_OUTPUT
+
+      - name: Check that PR satisfies requirements for cherry-pick action and notify if not
+        run: |
+          if [[ "${{ steps.pr_info.outputs.status }}" != "MERGED" ]] || [[ "${{ steps.pr_info.outputs.base_ref }}" != fasttrack/* ]]; then
+            gh pr comment ${{ github.event.issue.number }} \
+              --repo ${{ github.repository }} \
+              --body 'Cherry pick action is only enabled for merged pull requests to `fasttrack/*` branches'
+          fi
+
+  # Actual cherry-pick work for each target branch
+  cherry_pick_commit:
+    needs: pr_info
+    if: |
+      needs.pr_info.outputs.status == 'MERGED' &&
+        startsWith(needs.pr_info.outputs.base_ref, 'fasttrack/')
+
+    runs-on: ubuntu-latest
+    permissions:
+      contents: write
+      pull-requests: write
+    defaults:
+      run:
+        shell: bash
+    name: Cherry pick
+    steps:
+      - name: Workflow trigger checkout
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+
+      - name: Validate and extract cherry pick target from comment
+        run: |
+          pr_comment="${{ github.event.comment.body }}"
+          branches_list=${pr_comment#'/cherry-pick '}
+          IFS=", " read -ra target_branches <<< "$branches_list"
+          echo "cherry_pick_target_branches=${target_branches[*]}" >> $GITHUB_ENV
+
+      - name: Configure local git repo
+        run: |
+          git config --local user.email "$GIT_USER_EMAIL"
+          git config --local user.name "$GIT_USER_NAME"
+        env:
+          GIT_USER_EMAIL: "cblmargh@microsoft.com"
+          GIT_USER_NAME: "CBL-Mariner Servicing Account"
```

Do we need the `env`?
```suggestion
          git config --local user.email "cblmargh@microsoft.com"
          git config --local user.name "CBL-Mariner Servicing Account"
```